### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Alexandre Philibert

### DISCUSSION THREAD

**Discussion by AlexandrePhilibert** [heig-vd-dai-course/heig-vd-dai-course#111] [Show and tell]
[Link to the repository](https://github.com/AlexandrePhilibert/dai-course-docker-docker-compose)

![image](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/39675639/45b3548e-a889-423b-b0ae-fc5d01d86915)

**Comment by ludelafo** [comment]
Everything looks good to me, however, I would still add the `data` directory to the gitignore as well for privacy reasons. I wouldn't share all my gigabytes of files on Git as well.. Do you see what I mean?
